graphql_compiler/compiler/sqlalchemy_extensions.py

```diff
@@ -64,15 +66,31 @@ def print_sqlalchemy_query_string(query, dialect):
     Returns:
         string that can be ran using sqlalchemy.sql.text(result)
     """
+    supported_dialects = (PGDialect_psycopg2, MSDialect_pyodbc)
+    if isinstance(dialect, supported_dialects):
+        class BindparamCompiler(dialect.statement_compiler):
+            def visit_bindparam(self, bindparam, **kwargs):
+                # A bound parameter with name param is represented as ":param". However,
+                # if the parameter is expanding (list-valued) it is represented as
+                # "([EXPANDING_param])" by default. This is an internal sqlalchemy
+                # representation that is not understood by databases, so we explicitly
+                # make sure to print it as ":param".
+                return f":{bindparam.key}"
+
+        return str(BindparamCompiler(dialect, query).process(query))
+    else:
```

Comment:
nit: would you mind putting the shorter clause first, for the sake of readability?